schema/cassandra/cadence/schema.cql

```diff
@@ -418,6 +421,7 @@ CREATE TABLE tasks (
 
 -- this table is only used for storage of mapping of domain uuid to domain name
 CREATE TABLE domains (
+  created_time       timestamp,
```

Comment:
Mind adding an updated timestmpa too?

Reply:
Discussed offline: 
Since domains don't have update operations, it doesn't matter if we add it or not